src/routes/index.svelte

```diff
@@ -1,7 +1,8 @@
 <script>
 	import P5 from '$lib/P5.svelte';
+	import RangeSlider from "svelte-range-slider-pips";
 
-	let branchHeight = 120;
+	let branchHeight = [150];
```

Comment:
[nit] guessing the array wrapper is a remnant from playing around with the sketch - can we convert back to an int?